### ISSUE TITLE
Introduce new API to the video widget

### ISSUE THREAD

**Issue by patrykmol** [eclipsesource/tabris-js#808]
_UPDATE: THE API WILL NOT REFLECT THE PROPOSAL HERE_

New API:

```
Properties:
speed
currentTime

Events:
playbackstatechanged 
-values: Unknown, ReadyToPlay, Failed, PlaybackDidFinish, PlaybackStalled
```

One thing that is missing is a way of enabling/disabling default controls (this most likely will be constructor-only flag).

**Comment by molasaheb** [comment]
I also need the autoplay property for Android. is this possible or I should look for alternatives?
thanks

**Comment by gosia-eclipsesource** [comment]
@molasaheb You will be able to start and stop videos and listen to events like "video finished playback", so you can play another video immediately after. Is that what you're looking for?

**Comment by mpost** [comment]
The new api has been released with the 1.10 release
